resources/l10n/de/LC_MESSAGES/loot.po

```diff
@@ -815,4 +817,164 @@ msgstr "Fehler: Konnte Protokollierung nicht initialisieren."
+"An error occurred while parsing the metadata list(s): %1%.\n"
+"\n"
+"Try updating your masterlist to resolve the error. If the error is with your "
+"user metadata, this probably happened because an update to LOOT changed its "
+"metadata syntax support. Your user metadata will have to be updated "
+"manually.\n"
+"\n"
+"To do so, use the 'Open Debug Log Location' in LOOT's main menu to open its "
+"data folder, then open your 'userlist.yaml' file in the relevant game "
+"folder. You can then edit the metadata it contains with reference to the "
+"documentation, which is accessible through LOOT's main menu.\n"
+"\n"
+"You can also seek support on LOOT's forum thread, which is linked to on "
+"[LOOT's website](https://loot.github.io/)."
+msgstr ""
+"Ein Fehler ist aufgetreten beim Einlesen von einer oder mehrerer "
+"Metadaten-Listen: %1%.\n"
+"\n"
+"Versuche die Masterliste zu aktualisieren um den Fehler zu beheben. "
+"Falls der Fehler Ihre benutzerdefinierten Metadaten betrifft, ist "
+"diese vermutlich aufgetreten weil ein LOOT Update die Unterstützung von "
+"Metadatensyntax verändert hat. Ihre benutzerdefinierten Metadaten müssen "
+"manuell aktualisiert werden.\n"
+"\n"
+"Um dies zu tun, nutze 'Öffne Debug-Log Pfad' in LOOTs Hauptmenü um "
+"den Ordner der Daten zu öffnen, öffnen Sie dann in relevanten Spiele-Ordner "
+"Ihre 'userlist.yaml' Datei. Sie können dann die beinhaltenden Metadaten "
+"bearbeiten mithilfe der Referenz zu der Dokumentation, welche.über "
+"LOOTs Hauptmenü erreichbar ist.\n"
+"\n"
+"Sie können auch nach Unterstützung in LOOTs Forumthema suchen, welche unter "
+"[LOOTs Webseite](https://loot.github.io/) verlinkt ist."
+
+#: src/gui/state/game/helpers.cpp:77
+msgid "%1% ITM record"
+msgid_plural "%1% ITM records"
+msgstr[0] "%1% ITM-Eintrag"
+msgstr[1] "%1% ITM-Einträge"
+
+#: src/gui/state/game/helpers.cpp:81
+msgid "%1% deleted reference"
+msgid_plural "%1% deleted references"
+msgstr[0] "%1% gelöschte Referenz"
+msgstr[1] "%1% gelöschte Referenzen"
+
+#: src/gui/state/game/helpers.cpp:87
+msgid "%1% deleted navmesh"
+msgid_plural "%1% deleted navmeshes"
+msgstr[0] "%1% gelöschter Navmesh"
+msgstr[1] "%1% gelöschte Navmeshes"
+
+#: src/gui/state/game/helpers.cpp:97
+msgid "%1% found %2%, %3% and %4%."
+msgstr "%1% hat %2%, %3% und %4% gefunden."
+
+#: src/gui/state/game/helpers.cpp:103
+msgid "%1% found dirty edits."
+msgstr "%1% hat schmutzige Bearbeitungen gefunden."
+
+#: src/gui/state/game/helpers.cpp:109 src/gui/state/game/helpers.cpp:115
+#: src/gui/state/game/helpers.cpp:120
+msgid "%1% found %2% and %3%."
+msgstr "%1% hat %2% und %3% gefunden."
+
+#: src/gui/state/game/helpers.cpp:124 src/gui/state/game/helpers.cpp:127
+#: src/gui/state/game/helpers.cpp:130
+msgid "%1% found %2%."
+msgstr "%1% hat %2% gefunden."
+
+#: src/gui/state/game/helpers.cpp:210
+msgid "LOOT has detected that \"%1%\" is invalid and is now ignoring it."
+msgstr "LOOT hat festgestellt das \"%1%\ ungültig ist wird es ab sofort "
```

Comment:
There's a missing double quote here that's causing the Appveyor build to fail, it should be `\"%1%\"`.

Reply:
Oops.
Okay, done.